### PR TITLE
Adding initial model for use-of-force

### PR DESCRIPTION
**Pull request by louisenorris**
## What does this pull request do?

Adds models for use-of-force and required dependencies

## What is the intent behind these changes?

To produce diagrams for documentation purposes

Screenshot 2021-12-10 at 10 04 48

Screenshot 2021-12-10 at 09 05 07

Screenshot 2021-12-10 at 10 05 01

Screenshot 2021-12-10 at 10 03 15

### CODE REVIEW

File: src/main/kotlin/uk/gov/justice/hmpps/architecture/model/UseOfForce.kt [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+        setUrl("https://github.com/ministryofjustice/use-of-force")
+
+        CloudPlatform.kubernetes.add(this)
+      }
+
+      reportCreator = model.addPerson("Report creator user", "Prison staff who creates reports and provides statements").apply {
+        uses(useOfForceService, "Creates Use of Force reports")
+        coordinator = model.addPerson("Coordinator user", "Prison staff who can view all complete and in progress reports and statements across their caseload")
+        reviewer = model.addPerson("Reviewer user", "Prison staff who can review and maintain reports across their caseload")
+        involvedStaff = model.addPerson("Involved staff user", "Prison staff who provides statements")
```
Comment (sldblog) [inline src/main/kotlin/uk/gov/justice/hmpps/architecture/model/UseOfForce.kt:66]: These actors are defined but don't have any relationship to the system or containers. Is that intentional?

Comment (louisenorris) [reply src/main/kotlin/uk/gov/justice/hmpps/architecture/model/UseOfForce.kt]: Thank you, I've added these in

File: src/main/kotlin/uk/gov/justice/hmpps/architecture/model/UseOfForce.kt [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+
+        CloudPlatform.kubernetes.add(this)
+      }
+
+      reportCreator = model.addPerson("Report creator user", "Prison staff who creates reports and provides statements").apply {
+        uses(useOfForceService, "Creates Use of Force reports")
+      }
+
+      coordinator = model.addPerson("Coordinator user", "Prison staff who can view all complete and in progress reports and statements across their caseload").apply {
+        uses(useOfForceService, "Views Use of Force reports")
```
Comment (sldblog) [inline src/main/kotlin/uk/gov/justice/hmpps/architecture/model/UseOfForce.kt:67]: _not a blocker, suggestion_
It's usually useful for posterity to also state the intent of these users, eg. "views these reports to {some business purpose}".

**Review by sldblog — APPROVED** [review]
👍 Nice! Thanks!

**Review by mxco86 — APPROVED** [review]
LGTM - thank you!

File: src/main/kotlin/uk/gov/justice/hmpps/architecture/model/UseOfForce.kt (hunk 2 above)
Comment (louisenorris) [reply src/main/kotlin/uk/gov/justice/hmpps/architecture/model/UseOfForce.kt:67]: Thank you - I'll have a think about this

**Pull request merged**